src/hypofuzz/dashboard.py

```diff
@@ -273,6 +273,21 @@ def send_nowait_from_anywhere(msg: ListenerEventT) -> None:
             send_channel.send_nowait(msg)
 
     db = get_db()
+    # load initial database state before starting dashboard
+    for key in db.fetch(b"hypofuzz-test-keys"):
+        reports = db.fetch_reports(key)
+        metadata = db.fetch_metadata(key)
+        if reports:
+            # TODO we should really add the node id to hypofuzz-test-keys entries
+            node_id = reports[0]["nodeid"]
+            REPORTS[node_id] = SortedList(reports, key=lambda r: r["elapsed_time"])
```

Comment:
I can pretty easily construct cases where there are multiple `nodeid`s for a single DB key, or vice-versa.  They're not tests that I'd expect anyone to write naturally and so I'm not proposing to try 'fixing' that, but I do want to treat database keys as the underlying source of truth and nodeids (which are also less stable!) as only a presentation-layer thing.